src/migrations/add-harvard-data/1682097649111-AddHarvardData.ts

```diff
@@ -0,0 +1,106 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import * as csvParser from 'csv-parser';
+import * as fs from 'fs';
+
+export class AddHarvardData1682097649111 implements MigrationInterface {
```

Comment:
Please use the name `USPresident1976–2020` and also add a README file with description.
https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/42MVDX

Reply:
Done